easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = "PythonBundle"
+
+name = 'matlab-proxy'
+version = '0.5.4'
+
+homepage = 'https://github.com/mathworks/matlab-proxy'
+description = 'MATLAB Integration for Jupyter'
```

Comment:
`matlab-proxy` is independent from Jupyter, please update the description.

Reply:
I blame @Micket who made the original one :-) I didn't even check the description :-)

Fixed

Reply:
I agree, in case of doubt always blame @Micket :laughing:

Reply:
The *jupyter*-matlab-proxy is actually a jupyter thing! Though i'm actually unsure in what way. There doesn't seem to be any difference at all between these two.

Reply:
My understanding is that `matlab-proxy` does not provide the integration with jupyter. It is just the http proxy to the web server of MATLAB. `jupyter-matlab-proxy` is matlab-proxy + jupyter-server-proxy to have that running inside JupyterLab.